Godeps/Godeps.json

```diff
@@ -12,58 +13,122 @@
 		},
 		{
 			"ImportPath": "github.com/amir/raidman",
-			"Rev": "2c924ed306089881ad89210d6ec1ada34f803403"
+			"Rev": "91c20f3f475cab75bb40ad7951d9bbdde357ade7"
+		},
+		{
+			"ImportPath": "github.com/amir/raidman/proto",
+			"Rev": "91c20f3f475cab75bb40ad7951d9bbdde357ade7"
 		},
 		{
 			"ImportPath": "github.com/golang/protobuf/proto",
-			"Rev": "5677a0e3d5e89854c9974e1256839ee23f8233ca"
+			"Rev": "7cc19b78d562895b13596ddce7aafb59dd789318"
 		},
 		{
 			"ImportPath": "github.com/gopherjs/gopherjs/js",
 			"Rev": "4b53e1bddba0e2f734514aeb6c02db652f4c6fe8"
 		},
 		{
 			"ImportPath": "github.com/intelsdi-x/snap/control/plugin",
-			"Comment": "v0.11.0-beta",
-			"Rev": "42d9567651258bba268d94701eb171458e459645"
+			"Comment": "v0.10.0-beta-280-g891c09f",
```

Comment:
You need to check your snap directory it's using to generate these versions. It's not picking up the latest tags in the repo.